understanding/label-in-name.html

```diff
@@ -58,210 +56,129 @@
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 2.5.3:</span>Label in Name (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 2.5.3:</span>ラベルを含む名前 (name) (レベル A)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>For <a href="#dfn-user-interface-component">user interface components</a> with <a href="#dfn-label">labels</a> that include <a href="#dfn-text">text</a> or <a href="#dfn-image-of-text">images of text</a>, the <a href="#dfn-name">name</a> contains the text that is presented visually.
+                  <p><a href="#dfn-user-interface-component">ユーザインタフェース コンポーネント</a>が<a href="#dfn-text">テキスト</a>又は<a href="#dfn-image-of-text">文字画像</a>を含む<a href="#dfn-label">ラベル</a>を持つ場合、視覚的に提示されたテキストが<a href="#dfn-name">名前 (name) </a> に含まれている。
                   </p>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>A best practice is to have  the text of the label at the start of the name.</p>
+                     <p class="note-title marker">注記</p>
+                     <p>ベストプラクティスは、ラベルのテキストを名前 (name) の最初に使用することである。</p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   			
-                  <dt>Goal</dt>
-                  <dd>The visual label for controls is a trigger for speech activation.</dd>
+                  <dt>目標</dt>
+                  <dd>コントロールの視覚的なラベルは、音声による起動のトリガーとなる。</dd>
                   			
-                  <dt>What to do</dt>
-                  <dd>Where practical, make the control’s text label and name match.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>可能な場合、コントロールのテキストラベルと名前とを一致させる。</dd>
                   			
-                  <dt>Why it's important</dt>
-                  <dd>People who operate with voice interaction use the visible labels in their commands.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>声による操作を行う人々は、コマンドで視覚的なラベルを使用する。</dd>
```

Comment:
visible label は「可視ラベル」と訳されているので、合わせてみました。

```suggestion
                  <dd>声による操作を行う人々は、可視ラベルをコマンドに使用する。</dd>
```